### ISSUE TITLE
Docup on ZOWE Prereqs?

### ISSUE THREAD

**Issue by josephperkins50** [zowe/community#1033]
The ZOWE Installation manual states that PERL and PYTHON are not required to be installed on Z/OS but ZOWE.ORG states that Z/OS requires the install.

If these are not required, please docup ZOWE.ORG.

Regards,
Joseph Perkins

**Comment by josephperkins50** [comment]
Are PYTHON and PERL required on Z/OS for the latest ZOWE Install?

**Comment by balhar-jakub** [comment]
@MichaelABroadcom This looks like a docs issue. Is it possible to move it to the docs-site repo?

**Comment by balhar-jakub** [comment]
@jackjia-ibm I believe that the answer to the actual question is that no, they are not needed.

**Comment by jackjia-ibm** [comment]
@balhar-jakub yes you are right. they are not needed for running Zowe runtime.

**Comment by MichaelABroadcom** [comment]
> @MichaelABroadcom This looks like a docs issue. Is it possible to move it to the docs-site repo?

I'll check into this.

**Comment by josephperkins50** [comment]
Hi Michael,

I'm a newbie. I'll be happy to move it to the doc-sites repo if you can give me directions in how to get there.

Thank you and
Regards,
Joseph Perkins (Contractor)
Sr. CA-IDMS Database Administrator, All in Solutions LLC
Office of Information and Technology (OIT), Development, Security, and Operations (DevSecOps)
Infrastructure Operations (IO)
Platform Support (PS)
Office: (530) 260-0165
GFE Mobile  (510) 599-6133
100% Remote Monday - Friday 9:00 am- 5:00pm CST
